code/modules/food_and_drinks/kitchen_machinery/cooker.dm

```diff
@@ -85,7 +85,7 @@
 
 /obj/machinery/cooker/proc/changename(obj/item/name, obj/item/setme)
 	setme.name = "[thiscooktype] [name.name]"
-	setme.desc = "[name.desc]. It has been [thiscooktype]"
+	setme.desc = "[name.desc] It has been [thiscooktype]."
```

Comment:
```suggestion
	setme.desc = "[name.desc], it has been [thiscooktype]."
```
I think this is better?

Reply:
If that comma were there, the example in OP would read "Delicious, but with a potentially overwhelming odor**.,** it has been deep fried."

Almost all food descriptions already end in punctuation, which is why I removed the unnecessary period from that line.